test/test/test.js

```diff
@@ -37,6 +37,80 @@ describe('Bridge', function () {
     );
   });
 
+  describe('DOT App', function () {
+
+    it('should transfer DOT from Substrate to Ethereum', async function () {
+
+      let amount = BigNumber('100000000000000'); // 100 DOT (12 decimal places in this environment)
+      let amountWrapped = BigNumber(Web3.utils.toWei('100', "ether")); // 100 SnowDOT (18 decimal places)
+
+      const account = ethClient.accounts[1];
+
+      let beforeEthBalance = await ethClient.getDotBalance(account);
+      let beforeSubBalance = await subClient.queryAccountBalance(polkadotSenderSS58);
+
+      // lock DOT using basic channel
+      await subClient.lockDOT(subClient.alice, account, amount.toFixed(), 0)
+      await sleep(70000);
```

Comment:
Is there a better way to wait for the event arriving on Ethereum than sleeping?

Reply:
Definitely, we can always update the E2E tests to watch for events. Its just very low-hanging fruit in the scheme of things.